### PR TITLE
[SL Alpha5] Add paylocity connector

### PR DESCRIPTION
**Pull request by Chuhaa**
## Purpose
$subject

## Security checks
 - Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines? yes
 - Ran FindSecurityBugs plugin and verified report? no
 - Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets? yes

## Test environment
JDK 11
Ballerina Alpha5
Ubuntu 20.04

### CODE REVIEW

File: openapi/paylocity/Ballerina.toml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+[package]
+license = ["Apache-2.0"]
+keywords = ["paylocity", "HR", "payroll"]
+org = "ballerinax"
+name = "payroll"
```
Comment (indikasampath2000) [inline openapi/paylocity/Ballerina.toml:5]: Here name must be `paylocity` not `payroll`

File: openapi/paylocity/client.bal [hunk 2]
```diff
@@ -0,0 +1,442 @@
+import  ballerina/http;
```
Comment (indikasampath2000) [inline openapi/paylocity/client.bal:1]: License header missing

File: openapi/paylocity/types.bal [hunk 3]
```diff
@@ -0,0 +1,1438 @@
+public type CompanyCodesArr CompanyCodes[];
```
Comment (indikasampath2000) [inline openapi/paylocity/types.bal:1]: License header missing

File: openapi/paylocity/Ballerina.toml (hunk 1 above)
Comment (Chuhaa) [reply openapi/paylocity/Ballerina.toml]: Fixed in https://github.com/ballerina-platform/ballerinax-openapi-connectors/commit/30ae73f063391071311ed2f7f5a9639bfe05a36c

File: openapi/paylocity/client.bal (hunk 2 above)
Comment (Chuhaa) [reply openapi/paylocity/client.bal:17]: Fixed in https://github.com/ballerina-platform/ballerinax-openapi-connectors/commit/30ae73f063391071311ed2f7f5a9639bfe05a36c

File: openapi/paylocity/types.bal (hunk 3 above)
Comment (Chuhaa) [reply openapi/paylocity/types.bal:17]: Fixed in https://github.com/ballerina-platform/ballerinax-openapi-connectors/commit/30ae73f063391071311ed2f7f5a9639bfe05a36c

**Review by indikasampath2000 — APPROVED** [review]

**Pull request merged**